src/index.ts

```diff
@@ -139,34 +141,74 @@ class Index<T> implements Types.IndexInterface<T> {
   /// INDEX
   ///
   /**
-   * Show index information.
+   * Get index information.
    * @memberof Index
-   * @method show
+   * @method getInfo
    */
-  async show(): Promise<Types.IndexResponse> {
+  async getInfo(): Promise<Types.IndexResponse> {
```

Comment:
```suggestion
  async getRawInfo(): Promise<Types.IndexResponse> {
```

Reply:
I would rename this method to make clear the information is raw. Because the `getInfo` method is really confusing with `fetchInfo` method by just reading the names of the methods. Not obvious or intuitive when we will come back to the code base haha

Reply:
Or another name, I just want the difference to be clear.